<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QDialog" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 233, 235);
</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="surteeTreeWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>First Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last Name</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Suretee to:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="transactionsTreeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>591</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Saved</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Withdrawn</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>loan Debit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>loan Credit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;History of transactions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="saveLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Savings</string>
   </property>
  </widget>
  <widget class="QLabel" name="interestLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="savingsLcdNumber">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="digitCount">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="loanLcdNumber">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="digitCount">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>570</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>560</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="interestLcdNumber">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="digitCount">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="loanLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>loan</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>View</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
